packages/swapper/src/swappers/thorchain/utils/cosmos/cosmosTxData.ts

```diff
@@ -58,7 +57,7 @@ export const cosmosTxData = async (input: {
     buyAsset,
     slippageTolerance,
     deps,
-    tradeFee: quote.feeData.tradeFee,
+    buyAssetTradeFeeUsd: quote.feeData.tradeFee,
```

Comment:
```suggestion
    buyAssetTradeFeeUsd: quote.feeData.buyAssetTradeFeeUsd,
```

Reply:
Actioned in https://github.com/shapeshift/lib/pull/1038/commits/deb7c95d42e6ce7accc9dd411351f123ea71c814.